src/mgos_barometer_bme280.c

```diff
@@ -33,6 +35,9 @@ bool mgos_barometer_bme280_detect(struct mgos_barometer *dev) {
     return false;
   }
 
+  LOG(LL_INFO, ("Detecting BMx280 %d, reg b is: %d", dev->i2caddr, val));
```

Comment:
I'm not convinced this is a useful logline for users. Perhaps make it LL_DEBUG ?